config/packages/test/validator.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+    validation:
+        not_compromised_password: false
```

Comment:
[This](https://symfony.com/doc/4.4/reference/configuration/framework.html#not-compromised-password) sounds interesting and a nice feature. I think this would be a good policy.

Reply:
we can create a ticket to check it out ;)